### ISSUE TITLE
Update MendeleyDesktop version to 2.125

### ISSUE THREAD

**Issue by yakushabb** [flathub/com.elsevier.MendeleyDesktop#57]
MendeleyDesktop appears to be using version 1.19.8.

There is a newer version of the app available on the [Mendeley download page](https://www.mendeley.com/download-reference-manager/linux), which uses the AppImage format.

Direct download link: https://static.mendeley.com/bin/desktop/mendeley-reference-manager-2.125.0-x86_64.AppImage

**Comment by Erick555** [comment]
This is _Mendeley Reference Manager_ which is new app that replaces discontinued _Mendeley Desktop_ . For the latter version 1.19.8 is [latest version](https://www.mendeley.com/release-notes/) (and  last).

Someone would need to publish the new app to flathub.

**Comment by ghost** [comment]
If the app is discontinued may I mark it as EOL?

@bbhtt

**Comment by bbhtt** [comment]
Sure

**Comment by bagustris** [comment]
@yakushabb , 
I want to share two points here regarding Mendeley Desktop (not manager).
- Mendeley Desktop is not going to EOL >> https://blog.mendeley.com/2025/07/09/mendeley-is-not-going-anywhere/
- Mendeley Reference Manager is rubbish, but Mendeley Desktop is not >> https://www.reddit.com/r/Mendeley/comments/10gfzoq/mendeley_reference_manager_is_a_horrible_program/

**Comment by ghost** [comment]
In that case, you can create a PR to update to a supported version. We can then remove the EOL status.

**Comment by Erick555** [comment]
They didn't seem to release any new version despite the announcement.

**Comment by bagustris** [comment]
Yes, the current version is still 1.9.8. It might be the final version, and they promise a security update if needed.

The previously available Mendeley in Flatpak is the only way to install Mendeley in modern Linux, since the deb/rpm will not work in recent Linux without any modification. I hit an error when opening Mendeley via Flatpak that I had previously installed, and tried reinstalling it, but the installation page was gone.

I've already solved my problem, and Mendeley works again in my flatpak run (the error was due to a hybrid GPU, Intel + Nvidia). I may propose a PR if allowed:
- fixing Mendeley’s bundled Qt 5.10 tries to use GLX, which fails under Flatpak on hybrid GPU setups.
- fixing EOL
